Directory.Build.targets

```diff
@@ -33,21 +37,12 @@
   </PropertyGroup>
 
   <PropertyGroup Condition="'$(Coverage)' == 'true'">
-    <!-- https://github.com/tonerdo/coverlet/issues/363 -->
-    <DeterministicSourcePaths>false</DeterministicSourcePaths>
-
-    <!-- https://github.com/tonerdo/coverlet/issues/618 -->
-    <IncludeTestAssembly>true</IncludeTestAssembly>
-
-    <CollectCoverage>true</CollectCoverage>
-    <SingleHit>true</SingleHit>
-    <CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
-    <CoverletOutputFormat>opencover</CoverletOutputFormat>
-    <CoverletOutput>$(ArtifactsDir)coverage\$(MSBuildProjectName)_$(TargetFramework)_$(_TestArchitecture).coverage</CoverletOutput>
-    <Include></Include>
-    <Exclude></Exclude>
-    <ExcludeByAttribute>ExcludeFromCodeCoverage</ExcludeByAttribute>
-    <ExcludeByFile></ExcludeByFile>
+    <!-- AzDO only supports coverage format but here we use codecov.io so we go with Cobertura format -->
+    <TestRunnerAdditionalArguments>$(TestRunnerAdditionalArguments) --collect:"Code Coverage;Format=Cobertura"</TestRunnerAdditionalArguments>
+    <!-- TODO: On next arcade bump of Microsoft.CodeCoverage/Microsoft.NET.Test.Sdk we will have to update this line to be netstandard2.0 -->
+    <!-- Because arcade is passing dlls directly to 'dotnet test' command, we need to manually provide path to coverage datacollector -->
+    <TestRunnerAdditionalArguments>$(TestRunnerAdditionalArguments) --test-adapter-path $(PkgMicrosoft_CodeCoverage)/build/netstandard1.0</TestRunnerAdditionalArguments>
+    <TestRunnerAdditionalArguments>$(TestRunnerAdditionalArguments) --settings roslyn-analyzers.runsettings</TestRunnerAdditionalArguments>
```

Comment:
Locally this is not working for me. Probably better:

<TestRunnerAdditionalArguments>$(TestRunnerAdditionalArguments) --settings $(RepoRoot)roslyn-analyzers.runsettings</TestRunnerAdditionalArguments>